cmd/terramate/cli/cli.go

```diff
@@ -1215,6 +1226,64 @@ func (c *cli) partialEval() {
 	}
 }
 
+func (c *cli) getConfigValue() {
```

Comment:
This is quite similar to cli.eval from the other PR, so I imagine it will be extracted in some way no ? (or Im missing some difference)

Reply:
there's some differences hard to extract, like this one works on the var traversals and serialization is different (has --as-json also).